src/meta/MetaServiceHandler.h

```diff
@@ -118,6 +118,9 @@ class MetaServiceHandler final : public cpp2::MetaServiceSvIf {
     folly::Future<cpp2::ListTagIndexesResp>
     future_listTagIndexes(const cpp2::ListTagIndexesReq& req) override;
 
+    folly::Future<cpp2::ExecResp>
+    future_rebuildTagIndex(const cpp2::RebuildIndexReq& req) override;
+
```

Comment:
Could we unify future_rebuildTagIndex and future_rebuildEdgeIndex to future_rebuildIndex?

Reply:
Seems we use it to check it's tag or edge